doc/api/replication.md

```diff
@@ -0,0 +1,106 @@
+# Thali Cordova Peer Communication API #
+
+## Methods:
+- `StartBroadcasting`
+- `StopBroadcasting`
+- `Connect`
+- `Disconnect`
+- `KillConnection`
+
+Events:
+- `peerAvailabilityChanged`
+- `networkChanged`
+
+***
+
+METHODS:
+
+***
+
+### `StartBroadcasting(deviceName, portNumber, callback)`
+
+This method starts advertising or broadcasting its availability at the following port that PouchDB/Express-PouchDB is listening.  
+
+#### Arguments:
+
+1. `deviceName` : `String` – the device name.
+2. `portNumber` : `Number` – obtained by the user of the code which indicates the port number of our local PouchDB/Express-PouchDB
```

Comment:
I would just specify a tcp/ip local listener. We want people to re-use our code for lots of scenarios, not just Express-PouchDB.